mibidata/tests/test_mibi_image.py

```diff
@@ -95,14 +112,36 @@ def test_set_channels_invalid_tuple(self):
         with self.assertRaises(ValueError):
             image.channels = invalid_tuple_3
 
-    def test_get_labels(self):
-        image = mi.MibiImage(TEST_DATA, STRING_LABELS)
-        self.assertEqual(image.labels, image.channels)
-
-    def test_set_labels(self):
-        image = mi.MibiImage(TEST_DATA, STRING_LABELS)
-        image.labels = TARGET_LABELS
-        self.assertEqual(image.channels, TARGET_LABELS)
+    @unittest.mock.patch('sys.stdout', new_callable=io.StringIO)
+    def test_convert_from_previous_metadata_versions(self, mock_stdout):
+        image = mi.MibiImage(TEST_DATA, TUPLE_LABELS)
+        image.point_name = OLD_METADATA['point_name']
+        image.folder = OLD_METADATA['folder']
+        image.version = OLD_METADATA['version']
+        image._convert_from_previous_metadata_versions()
+        message = ("WARNING! you are trying to use the old metadata "
```

Comment:
Single quotes in this file

Reply:
I'm using single quotes within the double quotes, so I'm keeping the double quotes. Let me know if you want another format for escaping the single quotes.